app/controllers/admin/members_controller.rb

```diff
@@ -22,6 +22,18 @@ def update
 
     private
 
+    def filter
+      @filter ||= Filter::Member.new(filter_params)
+    end
+
+    def filtered_users
+      User::Filtered.call(filter_params)
```

Comment:
I believe it's a good idea considering we may need filtering for something other than members